test/peer-file-watcher.js

```diff
@@ -331,7 +331,7 @@ function sendRequests(channel, count, cb) {
                     as: 'raw'
                 }
             }).send('echo', 'a', 'body', onResponse);
-        });
+        }, Math.random() * 1000);
```

Comment:
With batched setImmediate in node4 the test fires all the outbound requests in next tick instead of in the next N ticks.

I want to evenly spread out the socket allocations over N ticks of time.